android/quest/src/main/assets/configs/app/composition_config.json

```diff
@@ -153,6 +153,16 @@
           }
         }
       ]
+    },
+    {
+      "title": "ic_households",
+      "mode": "working",
+      "focus": {
+        "reference": "Binary/144175",
+        "identifier": {
+          "value": "ic_households"
+        }
+      }
```

Comment:
I suspect we will have so many icons in the app. Can they be grouped into a Composition section?

Reply:
We can